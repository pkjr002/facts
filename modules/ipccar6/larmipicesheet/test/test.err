ipccar6_postprocess_larmipicesheet_dev.py:56: RuntimeWarning: invalid value encountered in add
  waissamps = waissamps + pensamps
mv: cannot stat ‘*.nc’: No such file or directory
