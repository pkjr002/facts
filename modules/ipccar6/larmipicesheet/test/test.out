Initiating test in /scratch/rk509/test.1666640435...
Extracting data files...
Executing workflow...
python ipccar6_preprocess_larmipicesheet.py --pipeline_id icesheets-ipccar6-larmipicesheet-ssp585 --scenario ssp585 --baseyear 2005 --pyear_start 2020 --pyear_end 2100 --pyear_step 10 --tlm_data 1
python ipccar6_fit_larmipicesheet.py --pipeline_id icesheets-ipccar6-larmipicesheet-ssp585
python ipccar6_project_larmipicesheet_dev.py --pipeline_id icesheets-ipccar6-larmipicesheet-ssp585 --nsamps 20000 --seed 4321 --crateyear_start 2080 --crateyear_end 2100
python ipccar6_postprocess_larmipicesheet_dev.py --pipeline_id icesheets-ipccar6-larmipicesheet-ssp585 --chunksize 662 --locationfile test/location_list.lst
Collecting output files...
